.github/PULL_REQUEST_TEMPLATE/bugfix.md

```diff
@@ -0,0 +1,27 @@
+---
+name: BugFix PR
+about: Create a PR for a Bugfix
+title: 'ISSUE_NUMBER - Name'
+labels: ''
+assignees: ''
+
+---
+
+### Description
```

Comment:
Isn't description and previous behavior described in the Issues (which should be linked).
Even current behaviour should be there IMO.
This feels little like paperwork.

Reply:
A lot of times a bug/feature request is described in a sense of "I am not sure what and why is happening but this seems wrong and here is a proposition what should be". Then there might be a discussions both online and offline. Rather than to search for what was finally agreed upon and implemented I would have a sentence or two here.

How many of us go back to the issue and update the issue description?

Reply:
TL;DR version:
Limited value, introduces overhead

Long version:
Let me explain why I think this is counterproductive, well PR templates in general (until they pick-up commit log)
I think adding the fix to PR is the second worst place after not having it all. When one looks back when & what was changed, the usual first place are the commit logs, or eventually the original issue. PR are not very useful for that IMO.
I understand that the description here gives a kick-start to reviewer - that's why I like the autopickup of the commit log. But on the other hand for complicated PRs to make a competent review requires more effort and clicking the linked issue is comparably small deal. For easy ones it doesn't matter on the other hand, and the need to compose a description is an overhead.
Furthermore, if we use the documentation scripts you created, author should always return to the original Issue and a _Release notes_ entry there anyway.